main.css

```
@@ -84,3 +84,12 @@ input[type="number"] {
 		transform:rotate(10deg)
 	}
 }
+
+#error-message{
```

There's no opacity rule on the div so the transition might not be smooth 
```suggestion
#error-message{
     opacity: 0;
     animation: fadeIn 1s forwards;
}
```